types/advocates.ts

```diff
@@ -25,4 +25,19 @@ interface Advocate {
   slackUsername?: string;
 }
 
-export { AdvocatesWorldRegion, ADVOCATES_WORLD_REGION_OPTIONS, Advocate };
+interface QampProject {
+  name: string;
+  url: string;
+  website?: string;
+  tags: string[];
+  mentors?: string[];
+  mentees?: string[];
+  contributors?: string[];
```

Comment:
The should all be required, right?

An empty string would also be valid if, for instance, there are no extra contributors in a given project.

Also, I think "contributors" are called "collaborators" in our database. It might be better to reflect that here too.

Reply:
These are great questions! 

Are you able to see what the data looks like already? Maybe we can sync on what the mock data shape could look like.. 

This is why I was trying to ask if we can get a more ideal shape of the project object. 

(Remember, I was mentioning to you and @y4izus that I'm going off of assumptions in this initial PR, and it's a little difficult because I could potentially be implementing things that will just have to be undone 😬 )